### PR TITLE
implement clases BaseRobot/FlyingRobot/DeliveryDrone

### PR DESCRIPTION
**Pull request by MaxSEngineer**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -2,4 +2,69 @@ class Cargo:
...
-# write your code here
+
+class BaseRobot:
+    def_p_step = 1
+
+    def __init__(self, name: str,
+                 weight: int,
+                 coords: None | list = None
+                 ) -> None:
+        self.coords = coords if coords else [0, 0]
```
Comment (vsmutok) [inline app/main.py:13]: better to use `or` in this case

File: app/main.py [hunk 2]
```diff
@@ -2,4 +2,69 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def_p_step = 1
```
Comment (vsmutok) [inline app/main.py:7]: why you need it?
Also `def_p_step` not clear name

**Review by vsmutok — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (MaxSEngineer) [reply app/main.py:13]: Thanks, its really clear way

File: app/main.py (hunk 2 above)
Comment (MaxSEngineer) [reply app/main.py:7]: I thought that i can control value of default increment in methods 
I deleted this

**Pull request closed without merging**